src/runtime_src/xdp/profile/plugin/aie_profile/client/aie_profile.cpp

```diff
@@ -112,8 +112,11 @@ namespace xdp {
 
     // Get partition columns
     // NOTE: for now, assume a single partition
-    auto partitionCols = xdp::aie::getPartitionStartColumns(metadata->getHandle());
-    auto startCol = partitionCols.at(0);
+    uint8_t startCol = 0;
+#ifdef XDP_CLIENT_BUILD
```

Comment:
We don't need this macro dependency check here. The current file is only compiled for Client flow with XDP_CLIENT_BUILD defined.

Reply:
Agree, removed the usage.